tck/src/main/java/org/eclipse/microprofile/config/tck/CdiOptionalInjectionTest.java

```diff
@@ -66,5 +66,7 @@ public void testOptionalInjection() {
 
         Assert.assertTrue(optionalValuesBean.getStringValue().isPresent());
         Assert.assertEquals(optionalValuesBean.getStringValue().get(), "hello");
+
```

Comment:
You need to assert notExistingProperty as well.
Assert.assertFalse(optionalValuesBean.getNotExistingProperty().isPresent());

Reply:
The test already asserts that on line 65

Reply:
the test already asserts that on line 65

Reply:
ah. I see. I missed the existing line. Your test is to test the scenario where no converter is involved.